api/MediaKeySession.json

```diff
@@ -349,10 +349,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "52"
```

Comment:
Oops, yet another case of onfoo attributes being added later. I don't doubt the results though.